OpenTESArena/CMakeLists.txt

```diff
@@ -78,6 +78,10 @@ IF (WIN32)
     ADD_DEFINITIONS("-D_SCL_SECURE_NO_WARNINGS=1")
 ENDIF()
 
+# Copy over required files
+FILE(COPY ${CMAKE_SOURCE_DIR}/data DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
The destinations should be `${CMAKE_CURRENT_BINARY_DIR}`, not `${CMAKE_BINARY_DIR}`. Sorry for not making that clear in my comment. The current binary directory ties it to where the current `CMakeLists.txt` outputs files.